src/components/Layout/Header.jsx

```diff
@@ -1,4 +1,5 @@
 import styled from "styled-components";
+import Navbar from "../Navbar";
```

Comment:
- [ ] Use `@/components/Navbar`.
- [ ] Leave a line between library imports and custom component imports.

Example:
https://github.com/ChessBase-India/helpchess/blob/568f841ac617f8d7c46de51909bd2b59933263b5/src/pages/index.jsx#L1C1-L19C71